pages/settings/SettingsLog.qml

```diff
@@ -132,6 +132,20 @@ Rectangle {
             font.pixelSize: 18
             font.family: MoneroComponents.Style.fontRegular.name
             text: qsTr("Daemon log") + translationManager.emptyString
+            themeTransition: false
+            onColorChanged: {
+                console.log("color changed")
+                var flickableContentYBefore = flickable.contentY
+                var daemonLogText = consoleArea.text
+                consoleArea.clear();
+                if (MoneroComponents.Style.blackTheme) {
+                    var daemonLogTextSwitchedTextColor = daemonLogText.replace(/#000000/g, '#ffffff');
```

Comment:
If we want to do it perfectly we would also have to fix the different lime / green color on change.